src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -826,6 +827,37 @@ result = file.getBytes()
 resultMetadata.put("content.type","image/png")
 ----
 
+=== Job Results
+
+Job Results are composed of two things: *Job Result List* and *Job Result Map*.
```

Comment:
`Job Results are composed of two elements`

Reply:
thanks